tests/puppeteer/campaigns/functional/BO/customers/customers/01_filterAndQuickEditCustomers.js

```diff
@@ -186,7 +186,7 @@ describe('Filter And Quick Edit Customers', async () => {
       await this.pageObjects.customersPage.filterCustomers(
         'select',
         'active',
-        DefaultAccount.enabled ? 'Yes' : 'NO',
+        DefaultAccount.enabled ? 'Yes' : 'No',
```

Comment:
Why not using the `Yes` or `No`in the `filterCustomers` function?
So when you pass a boolean it consider it as `Yes/No`

Reply:
It's just another way to do it, No preferences.
Do you think its easier to maintain when 'using the Yes or No in the filterCustomer' ?

Reply:
depend on what the filterCustomer do.
If every time you think a boolean is passed to the function it must be convert to a string, go with it.
If it's really rare to have a real boolean, you can add an extra parameter late to don't convert the boolean into `yes|no`
You're also able to create a new method.
It's just to remove the ternary everywhere, it's considered as duplicate logic.

Reply:
Yes, it's always a yes / no, and I was thinking, if the value of options in select changed (for YES/NO for example), it will be easier to changed it in filterCustomers than all tests
Thanks for suggestion :+1:

Reply:
changes done, what do you think about it ?